packages/babel-preset/src/assets/isAssetUrl.ts

```diff
@@ -0,0 +1,3 @@
+export function isAssetUrl(value: string): boolean {
```

Comment:
I intentionally didn't cover this by tests directly as it will be changed/removed in #234.